model/post.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 const (
-	POST_SYSTEM_MESSAGE_PREFIX = "system_"
-	POST_DEFAULT               = ""
-	POST_SLACK_ATTACHMENT      = "slack_attachment"
-	POST_JOIN_LEAVE            = "system_join_leave"
-	POST_HEADER_CHANGE         = "system_header_change"
+	POST_SYSTEM_MESSAGE_PREFIX  = "system_"
+	POST_DEFAULT                = ""
+	POST_SLACK_ATTACHMENT       = "slack_attachment"
+	POST_JOIN_LEAVE             = "system_join_leave"
+	POST_HEADER_CHANGE          = "system_header_change"
+	POST_OUT_OF_CHANNEL_MENTION = "system_out_of_channel_mention"
```

Comment:
I thought it would be useful to have on the client-side so we could change the rendering/handling of these posts. I was planning on using it to add an "invite these users to the channel" link to the post on the client, but that needed enough additional UI that I thought it would be best to break it off into another ticket.